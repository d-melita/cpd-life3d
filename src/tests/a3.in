200 128 .5 1000
