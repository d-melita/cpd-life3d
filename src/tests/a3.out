1 585667 87
2 198117 20
3 123360 6
4 117152 0
5 116181 0
6 116832 0
7 116421 0
8 116559 0
9 116344 0
